### PR TITLE
Updates krypton

### PR DESCRIPTION
**Pull request by phil65**
krypton changes now allow to use sys.listitem properly.

### CODE REVIEW

File: context.extendedinfo.dialog/addon.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      <visible>StringCompare(ListItem.dbtype,movie) | StringCompare(ListItem.dbtype,tvshow) | StringCompare(ListItem.dbtype,season) | StringCompare(ListItem.dbtype,actor) | StringCompare(ListItem.dbtype,director)</visible>
+    </item>
+  </extension>
+  <extension point="kodi.addon.metadata">
+    <platform>all</platform>
+    <summary lang="en">Open ExtendedInfo dialog from context menu in library</summary>
+    <description lang="en">Open ExtendedInfo dialog from context menu in library. Needs script.extendedinfo installed and activated. Available for movies, tv shows and seasons currently.</description>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 3, 29 June 2007</license>
+    <forum></forum>
+    <source></source>
```
Comment (Razzeee) [inline context.extendedinfo.dialog/addon.xml]: No source link?

File: context.extendedinfo.ratemedia/addon.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      <visible>StringCompare(ListItem.dbtype,movie) | StringCompare(ListItem.dbtype,tvshow) | StringCompare(ListItem.dbtype,episode)</visible>
+    </item>
+  </extension>
+  <extension point="kodi.addon.metadata">
+    <platform>all</platform>
+    <summary lang="en">Rate media at TheMovieDB from context menu in library</summary>
+    <description lang="en">Rate media at TheMovieDB from context menu in library. Needs script.extendedinfo installed and activated. Available for movies, tv shows and episodes currently.</description>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 3, 29 June 2007</license>
+    <forum></forum>
+    <source></source>
```
Comment (Razzeee) [inline context.extendedinfo.ratemedia/addon.xml]: No source?

**Comment by tamland** [comment]
>krypton changes now allow to use sys.listitem properly.

^huh? It's been there since the beginning, isengard..

File: context.extendedinfo.dialog/addon.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+<addon id="context.extendedinfo.dialog" name="ExtendedInfo Script - Open dialog" version="1.0.1" provider-name="phil65">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="script.extendedinfo" version="4.5.0"/>
+  </requires>
+  <extension point="kodi.context.item" library="addon.py">
+    <item>
+      <label>32000</label>
+      <visible>StringCompare(ListItem.dbtype,movie) | StringCompare(ListItem.dbtype,tvshow) | StringCompare(ListItem.dbtype,season) | StringCompare(ListItem.dbtype,actor) | StringCompare(ListItem.dbtype,director)</visible>
+    </item>
```
Comment (tamland) [inline context.extendedinfo.dialog/addon.xml]: This syntax is deprecated fyi: http://kodi.wiki/view/Context_Item_Add-ons Since you're PRing >=jarvis only

Comment (tamland) [inline context.extendedinfo.dialog/addon.xml]: ..not that it's intend to be removed anytime soon. Krypton is still backwards compatible

**Comment by phil65** [comment]
@tamland you´re right, i used wrong wording. sys.listitem worked earlier too of course but it wasnt necessary to use it since in MediaWindows you could also use xbmc.getInfoLabel("Listitem.xxx").

File: context.extendedinfo.dialog/addon.xml (hunk 3 above)
Comment (phil65) [inline context.extendedinfo.dialog/addon.xml]: I see, didnt notice that change. Will fix it for next update.

**Comment by MartijnKaijser** [comment]
These discussions shouldn't take place on our repository as it's sole intention is to only store scripts.

**Comment by phil65** [comment]
Nah, some discussion is totally fine, as long as it stays connected to this PR.

**Comment by Razzeee** [comment]
I feel like Phil and it might also make things much clearer / helpful

**Comment by phil65** [comment]
Updated.

**Comment by learningit** [comment]
Looks good to me. Do you want me to merge or are you still accumulating changes?

**Comment by phil65** [comment]
Merge.

**Pull request merged**